### PR TITLE
Use npm install instead of npm update for package bootstrap

### PR DESCRIPTION
**Pull request by easel**
- Npm update behavior has changed in versions > ~5.0.0

### CODE REVIEW

File: sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala [hunk 1]
```diff
@@ -572,7 +572,7 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
             if (useYarn.value) {
               Yarn.run("install", "--non-interactive")(targetDir, log)
             } else {
-              Npm.run("update")(targetDir, log)
+              Npm.run("install", ".")(targetDir, log)
```
Comment (julienrf) [inline sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala]: What’s the benefit of the `.`?

Comment (easel) [reply sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala]: Just explicitness:

Comment (easel) [reply sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala]: That said, I glazed over the first line when I was reading that help the first time, so it's apparently documented behavior to look for package.json in the current folder.

**Comment by cquiroz** [comment]
Should this make it work on npm >= 5.x

**Comment by easel** [comment]
Yes, should resolve it.

**Pull request merged**